src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public enum Colors {

```

Comment:
```suggestion
public enum Color {

```

enum is a class, and we have to use singular in class names